browser/brave_features_browsertest.cc

```diff
@@ -0,0 +1,14 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
@darkdh I couldn't find a test file for features. If this test can be moved somewhere else, let me know.